Assets/LeapMotion/Core/Scripts/Streams/Abstract/StreamSkippingFilter.cs

```diff
@@ -0,0 +1,119 @@
+﻿using System;
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+namespace Leap.Unity {
+
+  public abstract class StreamSkippingFilter<T> : MonoBehaviour,
```

Comment:
What is the point of making this an abstract class?  It seems to have concepts that are only applicable for pose-like streams (and indeed, it seems the only implementation is of type pose)

Reply:
The other implementer is still in AppModules -- or rather, was used in Paint, and was used for StrokePoints, which have poses, but also have other data.

Reply:
Ah I see, fair enough